### PR TITLE
Rework follow sheet layout

### PR DESCRIPTION
**Pull request by bfollington**
Fixes https://github.com/subconsciousnetwork/subconscious/issues/827

Changed from custom `.fraction(0.33)` to `.medium` detent to support the smaller screen sizes (see issue). While I was at it I decided to remove the DID and put a preview in its place. I think this is a more economical use of space.

https://github.com/subconsciousnetwork/subconscious/assets/5009316/cfc4982c-d22b-4da7-b5fb-f1436b0010ab

### CODE REVIEW

File: xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift [hunk 1]
```diff
@@ -169,18 +169,20 @@ struct FollowUserSheet: View {
     var onDismissError: () -> Void
     
     var caption: String {
-        state.petnameFieldCaption ?? ""
+        state.petnameFieldCaption ?? "Choose a nickname"
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift:172]: Nit: should be `String(localized:)`

**Review by gordonbrander — APPROVED** [review]
LGTM with some comments

File: xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift [hunk 2]
```diff
@@ -169,18 +169,20 @@ struct FollowUserSheet: View {
...
+            Spacer()
+            
             if let user = state.user {
-                ProfilePic(pfp: user.pfp, size: .large)
-                
-                Text(user.did.did)
-                    .font(.caption.monospaced())
-                    .foregroundColor(.secondary)
-                    .multilineTextAlignment(.leading)
+                HStack(alignment: .center) {
```
[the remark this answers is not in the document]
Comment (bfollington) [reply xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift:180]: Fwiw, I agree the design here does not... spark joy. I am shooting for a quick fix but this UI deserves a full design pass. 

I'll play with it a bit more to try and dial it in, unfortunately there isn't a "as small as possible without scrolling" setting for sheet size (that I can find).

[the remark this answers is not in the document]
Comment (bfollington) [reply xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift:180]: Reflowed the layout slightly, I still think this needs more love but it's acceptable for now.

image

[the remark this answers is not in the document]
Comment (bfollington) [reply xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift:180]: Confirmed working on iPhone 13 mini

image

**Pull request merged**